README.md

```diff
@@ -46,6 +46,16 @@ del whatever you want in Windows 32/64
 You can combine OS tags and rm all the things!
 ```
 
+### Override detection settings for linux-based shells on Windows
+
+By default, run-script-os will detect cygwin/git bash as Windows. If you would rather your platform be detected as Linux under these environments:
+
+Set environment variable:
+
+```
+RUN_OS_WINBASH=true
```

Comment:
Minor:  My instinct when I saw this name was that it did the opposite....  Suggest:  `RUN_OS_WINBASH_IS_LINUX`

Otherwise looks good...

Reply:
Great point, I've made that change.